### PR TITLE
fix: alteration scripts

### PR DESCRIPTION
**Pull request by gao-sun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "start": "cd packages/core && NODE_ENV=production node .",
     "cli": "logto",
     "alteration": "logto db alt",
+    "//": "# `changeset version` won't run version lifecycle scripts, see https://github.com/changesets/changesets/issues/860",
+    "ci:version": "changeset version && pnpm -r version",
```
Comment (charIeszhao) [inline package.json:15]: nit: Why don't we put this command directly in the `release.yml`?

**Comment by charIeszhao** [comment]
Btw, what is the command to rename `next-12345-blah-blah.js` scripts to `version-tag-blah-blah.js`?

File: package.json (hunk 1 above)
Comment (gao-sun) [reply package.json:15]: it's a universal script

Comment (gao-sun) [reply package.json:15]: so we'd better to duplicate the original behavior instead of depending on workflow

**Comment by gao-sun** [comment]
> Btw, what is the command to rename `next-12345-blah-blah.js` scripts to `version-tag-blah-blah.js`?

See [this](https://github.com/logto-io/logto/blob/47a6915bc4726a83ac056c589899571092fea737/packages/schemas/package.json#L18)

**Pull request merged**